plutus-core/plutus-core/src/PlutusCore/Pretty/PrettyConst.hs

```diff
@@ -162,6 +162,9 @@ instance PrettyBy ConstConfig Data where
 instance PrettyBy ConstConfig Value.K where
     prettyBy config = prettyBy config . Value.unK
 
+instance PrettyBy ConstConfig Value.Quantity where
+    prettyBy config = prettyBy config . Value.unQuantity
+
 instance PrettyBy ConstConfig Value where
     prettyBy config = prettyBy config . Value.toList
```

Comment:
Not that it matters much, but these should be newtype-derivable.

Reply:
It would require data constructors to be in scope (they aren't).